modules/_labs/core/react/spec/type.snapshot.tsx

```diff
@@ -1,5 +1,6 @@
+/** @jsx jsx */
```

Comment:
Are we going to add this to every file or change the pragma in our configs? As long as the pragma is not set to `preserve` in the Typescript config, this should work for downstream consumers.

Reply:
This was done automatically by their linting rules via https://github.com/emotion-js/emotion/blob/master/packages/eslint-plugin-emotion/docs/rules/jsx-import.md. I was planning to update the configs so I can disable this rule, but haven't got there yet.

Reply:
Just tried this and while I was able to move the pragma to the tsconfig successfully, it caused some other problems. Also, I'm not sure how to make Emotion's `jsx` import global without babel so you still need to `import {jsx} from '@emotion/core';` in every file that `css` is used.

Reply:
Just to track on the PR, I decided not to set the `jsxFactory` in `tsc` because we only need the `jsx` pragma for select files that use the `css` prop (mainly stories). This is added automatically by the ESLint rule linked above for files using the `css` prop, so it's not too big of a deal to require in the files as necessary. Also there doesn't seem to be any way to avoid the `jsx` import with `tsc` in these cases.

The biggest reason for not adding the pragma to all files is because it [downgrades fragment node experience](https://github.com/emotion-js/emotion/issues/1156).